mmv1/products/redis/Cluster.yaml

```diff
@@ -189,10 +189,13 @@ examples:
       subnet_name: "my-subnet"
       network_name: "my-network"
       deletion_protection_enabled: 'true'
+      allow_fewer_zones_deployment: 'false'
```

Comment:
```suggestion
```
recommend not using variable if doesn't need to be different depending on environment

Reply:
i'll set this to true